onnxscript/rewriter/ort_fusions/skip_normalization.py

```diff
@@ -32,15 +32,55 @@ def _skip_normalization(op, input, skip, gamma, epsilon, stash_type):
     return normalized, skip_sum
 
 
-_rule = pattern.RewriteRule(
-    _skip_norm_pattern, _skip_normalization, matcher=pattern.SimplePatternMatcher
+_skip_rms_rule = pattern.RewriteRule(
+    _skip_rmsnorm_pattern, _skip_rms_normalization, matcher=pattern.SimplePatternMatcher
 )
 
-skip_normalization_rules = [_rule]
-normalization_rules = rms_normalization_rules + skip_normalization_rules
-normalization_ruleset = pattern.RewriteRuleSet(normalization_rules)
+skip_rms_normalization_rules = [_skip_rms_rule]
+skip_rms_normalization_ruleset = pattern.RewriteRuleSet(skip_rms_normalization_rules)
 
 
-def fuse_normalization(model):
-    count = normalization_ruleset.apply_to_model(model)
-    print(f"Normalization count: {count}")
+def _skip_layernorm_pattern(op, input, skip, gamma, beta, epsilon, stash_type):
```

Comment:
nit: rename the "layernorm" part to "layer_norm" to be consistent with other usage